share/picongpu/examples/LaserWakefield/include/picongpu/param/incidentField.param

```diff
@@ -250,6 +242,14 @@ namespace picongpu
                 static constexpr auto laguerreModes = floatN_X<numModes + 1>(1.0);
                 static constexpr auto laguerrePhases = floatN_X<numModes + 1>(0.0);
                 /** @} */
+
+                template<typename My = GaussianPulseParam>
```

Comment:
We typically use `T_` as default names for template arguments and not `My`. I would like to stick to that guideline.

Reply:
This was already part of the original implementation from @chillenzer. 
I agree with you. @psychocoderHPC should this change go in this PR as well?

Reply:
renamed `My` to `T_Self`

Reply:
I'm aware of this and I didn't stick to that convention because I see it less like a template parameter and more like hack/workaround for injecting the correct type.

I'm fine with changing it but if so, we should probably do it consistently everywhere and adopt that as the new convention.